tools/remote_build/windows.bazelrc

```diff
@@ -0,0 +1,2 @@
+build --test_tag_filters=-exclude_windows
```

Comment:
is there a way to autodetect bazel is running on windows without needing to specify that explicitly?

Reply:
Can we use this https://docs.bazel.build/versions/master/platforms.html#built-in-constraints-and-platforms ?

Reply:
platform serves a different purpose than what you're describing. This is just for Windows local build, a form of platform will probably be introduced when we enable RBE for Windows